File: openraveros_tutorials/src/loadscene.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import roslib; roslib.load_manifest('openraveros_tutorials') 
import rospy

from openraveros.srv import env_loadscene

if __name__ == "__main__":
    rospy.wait_for_service('openrave/env_loadscene')
    env_loadscene = rospy.ServiceProxy('openrave/env_loadscene', env_loadscene)
    env_loadscene(filename='data/lab1.env.xml',resetscene=1)
    

File: openraveros_tutorials/src/loadscene.cpp
#include <ros/ros.h>
#include <openraveros/env_loadscene.h>

int main(int argc, char **argv)
{
    ros::init(argc, argv, "openraveclient");
    ros::NodeHandle n;
    ros::ServiceClient client = n.serviceClient<openraveros::env_loadscene>("openrave/env_loadscene");
    openraveros::env_loadscene srv;
    srv.request.filename = "data/lab1.env.xml";
    srv.request.resetscene = 1;
    if( !client.call(srv) ) {
        ROS_ERROR("Failed to call service env_loadscene");
        return 1;
    }
    return 0;
}
